src/libs/SteamApi/src/achievements.cpp

```diff
@@ -1,5 +1,5 @@
 //============================================================================================
-#include "achievements.h"
+#include "achievements.hpp"
```

Comment:
we don't use hpp extension in the project. do we really want to introduce it?

Reply:
Yes, I think we do. And all headers should be normalized to hpp.
however, here I did that unintentionally. But I still do not see much problem considering the above.
Please, let me know if we want to avoid this for now.